src/Console/InstallCommand.php

```diff
@@ -133,6 +134,11 @@ protected function replaceEnvVariables(array $services)
             $environment .= "\nMEILISEARCH_HOST=http://meilisearch:7700\n";
         }
 
+        if (in_array('minio', $services)) {
+            $environment .= "\nMINIO_ROOT_USER=sail";
+            $environment .= "\nMINIO_ROOT_PASSWORD=password\n";
+        }
```

Comment:
Is it necessary to inject anything in the environment file at all? Why not just have default values in the docker-compose.yml file?

Reply:
Not required, no. I can move the sail/password defaults to the stub file.

Reply:
OK